lemur/domains/views.py

```diff
@@ -86,6 +86,7 @@ def get(self):
         return service.render(args)
 
     @validate_schema(domain_input_schema, domain_output_schema)
+    @operator_permission.require(http_exception=403)
```

Comment:
for some of our users, the self service nature of lemur is an important feature. We'd likely want this gated with a documented configuration property e.g, ADMIN_ONLY_DOMAIN_CREATION